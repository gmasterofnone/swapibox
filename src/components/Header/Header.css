.header {
  align-items: center;
  display: flex;
  justify-content: center;
  margin: 50px;
  margin: 50px 14% 50px 14%;
}

.logo {
  height: 50px; 
}

.music {
  position: absolute;
  background-color: transparent;
  border: 1px solid #feda4a;
  border-radius: 3px;
  color: #feda4a;
  top: 5px;
  left: 80%;
  width: 90px;
  height: 18px;
  float: right;
}

 @media screen and (max-width: 466px) {
  .header {
    margin: 35px;
  }

  .logo {
    width: 106%;
  }
 }
