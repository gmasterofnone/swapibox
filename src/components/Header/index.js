import React from 'react';
import { NavLink } from 'react-router-dom'
import PropTypes from 'prop-types';


import logo from '../Utils/assets/logo.svg'

import './Header.css';

const Header = ( { getCards }) => {
  
  return (
    <div className="header" >
    <NavLink to=''
        >
        <img 
        className='logo' 
        src={logo}
        alt='Swapi Box Logo'
        onClick={() => getCards('films')}
        />
        </NavLink>
    </div>
  )
}

Header.propTypes = {
  getCards: PropTypes.func
}

export default Header; 