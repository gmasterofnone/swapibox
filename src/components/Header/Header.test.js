/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import Header from '.';

describe('Header', () => {
  
  it('matches the snapshot', () => {
    let wrapper = shallow(<Header />);

    expect(wrapper).toMatchSnapshot();
  });

});