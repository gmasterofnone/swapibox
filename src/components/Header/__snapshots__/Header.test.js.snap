// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <NavLink
        activeClassName="active"
        aria-current="page"
        to=""
      >
        <img
          alt="Swapi Box Logo"
          className="logo"
          onClick={[Function]}
          src="logo.svg"
        />
      </NavLink>,
      "className": "header",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "activeClassName": "active",
        "aria-current": "page",
        "children": <img
          alt="Swapi Box Logo"
          className="logo"
          onClick={[Function]}
          src="logo.svg"
        />,
        "to": "",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Swapi Box Logo",
          "className": "logo",
          "onClick": [Function],
          "src": "logo.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <NavLink
          activeClassName="active"
          aria-current="page"
          to=""
        >
          <img
            alt="Swapi Box Logo"
            className="logo"
            onClick={[Function]}
            src="logo.svg"
          />
        </NavLink>,
        "className": "header",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "activeClassName": "active",
          "aria-current": "page",
          "children": <img
            alt="Swapi Box Logo"
            className="logo"
            onClick={[Function]}
            src="logo.svg"
          />,
          "to": "",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Swapi Box Logo",
            "className": "logo",
            "onClick": [Function],
            "src": "logo.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
